4주차/[Lv.3]_등굣길/주다현.js

```diff
@@ -0,0 +1,17 @@
+//런타임 에러가 나는 코드랍니다... 아직 해결하지 못했어요ㅠㅠㅠㅠ
+
+function solution(m, n, puddles) {
+  const arr = Array.from(Array(n), () => Array(m).fill(1));
+  puddles.forEach((v) => {
+    const [x, y] = v;
```

Comment:
```suggestion
  puddles.forEach(([y, x]) => {
```
<img width="649" alt="image" src="https://github.com/prgrms-web-devcourse/FEDC4-JSAlgorithm-study/assets/66409882/fc9aea3d-ffb6-4f85-96ee-f335bb1c48b9">

> 가장 왼쪽 위, 즉 집이 있는 곳의 좌표는 (1, 1)로 나타내고 가장 오른쪽 아래, 즉 학교가 있는 곳의 좌표는 (m, n)으로 나타냅니다.

문제에서 좌표가 `(n, m)`이 아닌 `(m, n)`으로 나타낸다고 했으므로 반대로 배열을 구조분해할당해야합니다!! 😆